src/main/java/seedu/duke/option/OptionList.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.option;
+
+import java.util.ArrayList;
+
+public class OptionList implements OptionListInterface {
+    private final ArrayList<OptionInterface> optionList;
```

Comment:
I think this one may have to be "options" as it is a collection. When we call an instance of the OptionList we can probably call that optionList

Reply:
Ok, I will take note of this and change them. Thanks for the explanation!